pandas/core/arrays/period.py

```diff
@@ -606,52 +605,9 @@ def to_timestamp(self, freq=None, how='start'):
         new_data = libperiod.periodarr_to_dt64arr(new_data.asi8, base)
         return DatetimeArrayMixin(new_data, freq='infer')
 
-    def _maybe_convert_timedelta(self, other):
```

Comment:
this is moved to the PeriodIndex because it is only used there?

Reply:
Correct.

Reply:
Good!